### DISCUSSION TITLE
Orchard Core module demo

### DISCUSSION THREAD

**Discussion by duxonline** [OrchardCMS/OrchardCore#15584] [General]
I have built a pretty cool application based on Orchard Core. I think it will benefit the wide community and would like to demo it in Orchard Core meeting. Do I need to register somewhere up front, or I can just join the meeting by clicking the link [here](https://docs.orchardcore.net/en/latest/docs/resources/meeting/)? Thank you.

**Comment by hishamco** [comment]
What is your demo? what features/modules are used? Is there a repo for it?

**Reply by duxonline** [reply]
Thank you @hishamco. Sorry for the late reply, for some reason, I didn't get any notifcation of your reply. Anyway, it's a turnkey eCommerce product, including a few modules. It allows non tech-savvy people to manage products, orders, customers, inventory, payments, shipping, taxe, etc, without knowing technical concepts like content fields, content type, or content items. 

I'm thinking of offering it as a free product to Orchard Core community. The repository is private right now. Depending on the feedback I receive from community, I can share the repo with more people.

**Comment by hishamco** [comment]
Also you can have a look to https://github.com/OrchardCMS/OrchardCore.Commerce

**Reply by duxonline** [reply]
I did look at OrchardCore.Commerce before. To me, it's more like a commerce tool or builing blocks for developers, which allows them to build an commerce solution using the blocks provided. 

My product focuses on store owners experiences, which allows them to run their eCommerce stores without knowing the technical details of Orchard Core. So it's a very different approach.
